.github/workflows/ci.yaml

```diff
@@ -10,6 +10,11 @@ jobs:
       matrix:
         os: [ubuntu-18.04, ubuntu-20.04, macos-latest]
         swift: [5.5, swift-DEVELOPMENT-SNAPSHOT-2022-01-09-a]
+        include:
+          - swift: 5.5
+            expectedVersionString: "Swift version 5.5 (swift-5.5-RELEASE)"
```

Comment:
```suggestion
            expected-version-string: "Swift version 5.5 (swift-5.5-RELEASE)"
```

Reply:
I would suggest sticking to GitHub's kebab case style.

Reply:
Good point, will update 👌